src/Init/System/IO.lean

```diff
@@ -1428,6 +1427,9 @@ standard input is exhausted.
 @[extern "lean_io_process_child_take_stdin"] opaque Child.takeStdin {cfg : @& StdioConfig} : Child cfg →
     IO (cfg.stdin.toHandleType × Child { cfg with stdin := Stdio.null })
 
+/-- Returns the operating system process id of the child process. -/
+@[extern "lean_io_process_child_pid"] opaque Child.pid {cfg : @& StdioConfig} : Child cfg → UInt32
```

Comment:
@TwoFX Since you reviewed `Process.getPID` before, let me at least highlight this addition to you. I didn't see any reason to make this one effectful as there (now) is no way to purely construct a `Child`